R/pkg/R/generics.R

```diff
@@ -462,12 +462,9 @@ setGeneric("describe", function(x, col, ...) { standardGeneric("describe") })
 #' @export
 setGeneric("drop", function(x, ...) { standardGeneric("drop") })
 
-#' @rdname dropduplicates
+#' @rdname dropDuplicates
 #' @export
-setGeneric("dropDuplicates",
-           function(x, colNames = columns(x)) {
-             standardGeneric("dropDuplicates")
-           })
+setGeneric("dropDuplicates", function(x, col, ...) { standardGeneric("dropDuplicates") })
```

Comment:
hmm - putting `col` in the generic means that it will always be required ? I'm not sure how the code path with no arguments will work then. I think we can leave the generic as `dropDuplicates(x, ...)` and then have a more specific implementation. (See `na.omit` for example)